### ISSUE TITLE
MissingPluginException on Accessibility Widget testing 

### ISSUE THREAD

**Issue by arashgm** [fluttercommunity/plus_plugins#689]
## System info

Issue occurs on: Widget Tests
Plugin name: connectivity_plus
Plugin version: 2.1.0

## Steps to Reproduce

on testing Accessibility on a Widget which use connectivity_plus , when i want to assert **textContrastGuideline**
with code: `await expectLater(this, meetsGuideline(textContrastGuideline));` MissingPluginException thrown

## Logs

The following MissingPluginException was thrown running a test:
MissingPluginException(No implementation found for method check on channel
dev.fluttercommunity.plus/connectivity)

When the exception was thrown, this was the stack:
#0      MethodChannel._invokeMethod (package:flutter/src/services/platform_channel.dart:156:7)
<asynchronous suspension>
<asynchronous suspension>
(elided one frame from package:stack_trace)
